library/src/main/java/io/github/douglasjunior/androidSimpleTooltip/SimpleTooltip.java

```diff
@@ -956,6 +962,15 @@ public Builder textColor(int textColor) {
             return this;
         }
 
+        public Builder typeFace(String fontPath) {
```

Comment:
Please, add a `java doc` for `public Builder typeFace` and `public Builder fontSize`.

This is necessary to keep the documentation up to date. https://javadoc.jitpack.io/com/github/douglasjunior/android-simple-tooltip/master-53d0c51d77-1/javadoc/index.html

(can only keep the English version)